boards/beagle/beagleconnect_freedom/beagleconnect_freedom.dts

```diff
@@ -43,18 +43,33 @@
 		};
 	};
 
+	/**
+	 * The BeagleConnect Freedom has an on-board antenna switch (SKY13317-373LF) used to select
+	 * the appropriate RF signal port based on the currently-used PHY.
+	 *
+	 * Truth table:
+	 *
+	 * Path         DIO29 DIO30
+	 * ===========  ===== =====
+	 * Off          0     0
+	 * Sub-1 GHz    0     1     // DIO30 mux to IOC_PORT_RFC_GPO0 for auto
+	 * 20 dBm TX    1     0     // DIO29 mux to IOC_PORT_RFC_GPO3 for auto
+	 */
+	antenna_mux0: antenna_mux0 {
+		compatible = "skyworks,sky13317";
+		status = "okay";
+		gpios = <&gpio0 29 GPIO_ACTIVE_HIGH>, <&gpio0 30 GPIO_ACTIVE_HIGH>;
+		pinctrl-0 = <&board_ant_tx_pa_off &board_ant_subg_off>;
+		pinctrl-1 = <&board_ant_tx_pa_off &board_ant_subg_on>;
+		pinctrl-2 = <&board_ant_tx_pa_on &board_ant_subg_on>;
```

Comment:
When PA is on, shouldn't SubG be off like what you have in the comments above this statement?!?

Reply:
No, it is SUBG PA, as shown in the name, not just PA. The table shows the pin states, not the pin configuration. 

It is done this way since for launchxl it is also possible to have 2.4G PA using the switch and thus it has 4 pinctrl compared to our 3.

Reply:
Agree, this is correct but I understand where the confusion comes from. I think we should make this easier to understand for casual readers, see my comment in that respect.